doc/API/centreon-api-v2.yaml

```diff
@@ -2784,9 +3114,10 @@ components:
       properties:
         is_forced:
           type: boolean
-          required: false
           description: "Indicates whether the check is forced (do not take into account the check timeperiod)"
           example: true
+      required:
```

Comment:
This is not required anymore : https://github.com/centreon/centreon/pull/8854/files#diff-e32a83c866e6a6b0e3ea466fc26db9a1L2787